# Problem Statement 1:

In each of the following situations, state whether it is a correctly stated hypothesis testing problem and why? 
 
1. 𝐻0: 𝜇 = 25, 𝐻1: 𝜇 ≠ 25 <br>
2. 𝐻0: 𝜎 > 10, 𝐻1: 𝜎 = 10 <br>
3. 𝐻0: 𝑥  = 50, 𝐻1: 𝑥  ≠ 50 <br>
4. 𝐻0: 𝑝 = 0.1, 𝐻1: 𝑝 = 0.5 <br>
5. 𝐻0: 𝑠 = 30, 𝐻1: 𝑠 > 30 <br>

###### Solution:

1. 𝐻0: 𝜇 = 25, 𝐻1: 𝜇 ≠ 25 
        
Correctly stated as null hypothesis is stated as no difference between the parameter(𝜇) and it's estimate.

**************************************************************

2. 𝐻0: 𝜎 > 10, 𝐻1: 𝜎 = 10
        
Incorrectly stated as alternate hypothesis is stated as no difference between the parameter(𝜎) and it's estimate.

***************************************************************

3. 𝐻0: 𝑥  = 50, 𝐻1: 𝑥  ≠ 50
        
Incorrectly stated as null hypothesis is stated as no difference between the statistic(x) and it's estimate.

***************************************************************

4. 𝐻0: 𝑝 = 0.1, 𝐻1: 𝑝 = 0.5
        
Incorrectly stated as both null and alternate hypothesis is stated as no difference between the parameter(p) and it's estimate.

****************************************************************

5. 𝐻0: 𝑠 = 30, 𝐻1: 𝑠 > 30
        
Incorrectly stated as null hypothesis is stated as no difference between the statistic(s) and it's estimate.

# Problem Statement 2: 
 
The college bookstore tells prospective students that the average cost of its textbooks is Rs. 52 with a standard deviation of Rs. 4.50. A group of smart statistics students thinks that the average cost is higher. To test the bookstore’s claim against their alternative, the students will select a random sample of size 100. Assume that the mean from their random sample is Rs. 52.80. Perform a hypothesis test at the 5% level of significance and state your decision

###### Solution:

We are going to conduct one tailed z-test.

###### Step-1: State the hypothesis

H0:  μ = 52
    
Ha:  μ > 52

###### Step-2: Determine the significance level

The given significance level is 0.05.

###### Step-3: Define the decision Rule

Reject H0 if calculated p-value is less than 0.05.

###### Step-4: Calculate the z statistics

In [1]:
population_mean = 52
Std = 4.50
N = 100
sample_mean = 52.80
alpha = 0.05

import math

z = (sample_mean - population_mean)/(Std/math.sqrt(100))
print("z score:", z)

z score: 1.7777777777777715


###### Step-5: Find the p value

In [2]:
#from z table 
pvalue = 1 - 0.9616
print("pvalue:",pvalue)

pvalue: 0.03839999999999999


###### Step-6: Conclusion

for one tail test, since pvalue < 0.05, we reject the null hypothesis.

Therefore, we may conclude that the average cost of textbooks is higher that Rs. 52. 

# Problem Statement 3:  
 
A certain chemical pollutant in the Genesee River has been constant for several years with mean μ = 34 ppm (parts per million) and standard deviation σ = 8 ppm. A group of factory representatives whose companies discharge liquids into the river is now claiming that they have lowered the average with improved filtration devices. A group of environmentalists will test to see if this is true at the 1% level of significance. Assume \ that their sample of size 50 gives a mean of 32.5 ppm. Perform a hypothesis test at the 1% level of significance and state your decision. 

###### Solution:

We are going to perform one tailed z-test.

###### Step-1: State the hypothesis

H0:  μ = 34
    
Ha:  μ < 34

###### Step-2: Determine the significance level

The given significance level is 0.01.

###### Step-3: Define the decision Rule

Reject H0 if calculated p-value is less than 0.01.

###### Step-4: Calculate the z statistics

In [3]:
population_mean = 34
std = 8
N = 50
sample_mean = 32.5
alpha = 0.01

import math

z = (sample_mean - population_mean)/(std/math.sqrt(100))
print("z score:", z)

z score: -1.875


###### Step-5: Find the p value

In [4]:
#from z table 
pvalue = 0.0307
print("pvalue:",pvalue)

pvalue: 0.0307


###### Step-6: Conclusion

for one tail test, since pvalue > 0.01, we fail to reject the null hypothesis.

Therefore, we may conclude that that claim that companies have lowered the discharge of chemical pollutant is not true.

# Problem Statement 4:  
 
Based on population figures and other general information on the U.S. population, suppose it has been estimated that, on average, a family of four in the U.S. spends about $1135 annually on dental expenditures. Suppose further that a regional dental association wants to test to determine if this figure is accurate for their area of country. To test this, 22 families of 4 are randomly selected from the population in that area of the country and a log is kept of the family’s dental expenditure for one year. The resulting data are given below. Assuming, that dental expenditure is normally distributed in the population, use the data and an alpha of 0.05 to test the dental association’s hypothesis. 
 
1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994 

###### Solution:

We are going to perform two tailed t-est.

###### Step-1: State the hypothesis

H0:  μ = 1135
    
Ha:  μ != 1135

###### Step-2: Determine the level of significance

The level of significance is 5%.

###### Step-3: Find the critical value

df = 22-1 = 21

for 2 tailed test from t table,

lower tc = -2.080<br>
upper tc = 2.080

###### Step-4: Define the decision Rule

Reject H0 if calculated t value is not between -2.080 and 2.080.

###### Step-5: Calculate the t-stat

In [5]:
import statistics as stats
import scipy
import math

data = [1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994]

sample_mean = stats.mean(data)
std = stats.stdev(data)
population_mean = 1135
N = len(data)

t = (sample_mean - population_mean)/(std/math.sqrt(N))
print("t stat:", t)

t stat: -2.023137479931484


###### Step-6: Conclusion

Since calculated t value is between -2.080 and 2.080 , We fail to reject the null hypothesis.

Therefore we may conclude that the average anual dental expenditures is not $1135.

# Problem Statement 5:  
 
In a report prepared by the Economic Research Department of a major bank the Department manager maintains that the average annual family income on Metropolis is \\$48,432. What do you conclude about the validity of the report if a random sample of 400 families shows and average income of \\$48,574 with a standard deviation of 2000? 

###### Solution:

We are going to conduct two tailed z-test.

###### Step-1: State the hypothesis

H0: μ = $48,432
    
Ha: μ != $48,432

###### Step-2: Determine the level of significance

Assume the level of significance is 5%.

###### Step-3: Define the decision Rule

Reject H0 if calculated p-value is less than 0.05.

###### Step-4: Calculate the z statistics

In [6]:
population_mean = 48432
std = 2000
N = 400
sample_mean = 48574
alpha = 0.05

import math

z = (sample_mean - population_mean)/(std/math.sqrt(N))
print("z score:", z)

z score: 1.42


###### Step-5: Find the p value

In [7]:
#from z table
#P(Z<1.42) = 0.9222
#P(Z>1.42) = 1 - 0.9222 = 0.0778

#for 2 tailed test 
pvalue = 0.0778*2
print("pvalue:", pvalue)

pvalue: 0.1556


###### Step-6: Conclusion

Since pvalue > 0.05, we fail to reject the null hypothesis.

Therefore, we may conclude that the claim that average annual family income on Metropolis is $48,432 is true.

# Problem Statement 6: 
 
Suppose that in past years the average price per square foot for warehouses in the United States has been \\$32.28. A national real estate investor wants to determine whether that figure has changed now. The investor hires a researcher who randomly samples 19 warehouses that are for sale across the United States and finds that the mean price per square foot is \\$31.67, with a standard deviation of \\$1.29. assume that the prices of warehouse footage are normally distributed in population. If the researcher uses a 5% level of significance, what statistical conclusion can be reached? What are the hypotheses? 

###### Solution:

since the sample size is less than 30, We are going to conduct two taild t-test.

###### Step-1: State the hypothesis

H0: μ = $32.28
    
Ha: μ != $32.28

###### Step-2: Determine the level of significance

The given level of significance is 5%.

###### Step-3: Find the critical value

degree fd freedom, 
df = 19-1 = 18

for df 18 and level of significance 0.05, 

lower tc = -2.101<br>
upper tc = 2.101

###### Step-4: Define the decision Rule

Reject H0 if t < -2.101 or is t > 2.101.

###### Step-5: Calculate the t statistics

In [8]:
population_mean = 32.28
sample_mean = 31.67
std = 1.29
N=19
alpha = 0.05

import math

t = (sample_mean - population_mean)/(std/math.sqrt(N))
print("t-score:", t)

t-score: -2.06118477175179


###### Step-6: Conclusion

Since calculated t value is between -2.101 and 2.101 , We fail to reject the null hypothesis for sample size 19.

Therefore,we may conclude that there is no sufficient evidence to believe that the average price per square foot has changed.

# Problem Statement 7: 
 
Fill in the blank spaces in the table and draw your conclusions from it. 

###### Solution:

###### Calculations are as following:

### Type I Error:

The probability of making a type I error is denoted by α:

α = P(type I error)<br>
OR<br>
α = P(reject H0 when H0 is true)

###### α at μ=50

In [9]:
import math

# for 1st test
μ = 50
std = 2.5
n = 10

# alpha1 = P(X¯ < 48.5) + P(X¯ > 51.5)
z11 = (48.5-50)/(2.5/math.sqrt(10))
z12 = (51.5-50)/(2.5/math.sqrt(10))
print("Test1=>")
print("z11:",z11)
print("z12:",z12)

# alpha1 = P(Z < −1.90) +P(Z > 1.90)
alpha1 = 0.0287 + 0.0287
print("alpha1:", alpha1)
print("-"*30)

###################################################################

# for 2nd test
population_mean = 50
std = 2.5
n = 10

# alpha2 = P(X¯ < 48) + P(X¯ > 52)
z21 = (48-50)/(2.5/math.sqrt(10))
z22 = (52-50)/(2.5/math.sqrt(10))
print("Test2=>")
print("z21:",z21)
print("z22:",z22)

# alpha2 = P(Z < −2.53) +P(Z > 2.53)
alpha2 = 0.0057 + 0.0057
print("alpha2:", alpha2)
print("-"*30)

###################################################################

# for 3nd test
population_mean = 50
std = 2.5
n = 16

# alpha3 = P(X¯ < 48.81) + P(X¯ > 51.9)
z31 = (48.81-50)/(2.5/math.sqrt(16))
z32 = (51.9-50)/(2.5/math.sqrt(16))
print("Test3=>")
print("z31:",z31)
print("z32:",z32)

# alpha3 = P(Z < -1.90) + P(Z > 3.04)

alpha3 = 0.0287 + 0.0012
print("alpha3:", alpha3)

print("-"*30)

###################################################################

# for 4th test
population_mean = 50
std = 2.5
n = 16

# alpha4 = P(X¯ < 48.42) + P(X¯ > 51.58)
z41 = (48.42-50)/(2.5/math.sqrt(16))
z42 = (51.58-50)/(2.5/math.sqrt(16))
print("Test4=>")
print("z41:",z41)
print("z42:",z42)

# alpha4 = P(Z < −2.53) +P(Z > 2.53)
alpha4 = 0.0057 + 0.0057
print("alpha4:", alpha4)

Test1=>
z11: -1.8973665961010278
z12: 1.8973665961010278
alpha1: 0.0574
------------------------------
Test2=>
z21: -2.5298221281347035
z22: 2.5298221281347035
alpha2: 0.0114
------------------------------
Test3=>
z31: -1.9039999999999964
z32: 3.039999999999998
alpha3: 0.0299
------------------------------
Test4=>
z41: -2.5279999999999974
z42: 2.5279999999999974
alpha4: 0.0114


### Type II Error:

The probability of making a type I error is denoted by β:

β = P(type II error)<br>
OR<br>
β  = P(fail to reject H0 when H0 is false)

###### β at μ=52

In [10]:
import math

# for 1st test
μ = 52
std = 2.5
n = 10

# beta1 = P(48.5 ≤ X¯ ≤ 51.5)
z11 = (48.5-52)/(2.5/math.sqrt(10))
z12 = (51.5-52)/(2.5/math.sqrt(10))
print("Test1=>")
print("z11:",z11)
print("z12:",z12)

# beta1 = P(−4.43 ≤ Z ≤ −0.63) = P(Z ≤ −0.63)−P(Z ≤ −4.43)
beta1 = 0.2643 - 0
print("beta1:", beta1)
print("-"*30)

###################################################################

# for 2nd test
population_mean = 52
std = 2.5
n = 10

# beta2 = P(48 ≤ X¯ ≤ 52)
z21 = (48-52)/(2.5/math.sqrt(10))
z22 = (52-52)/(2.5/math.sqrt(10))
print("Test2=>")
print("z21:",z21)
print("z22:",z22)

# beta2 = P(−5.06 ≤ Z ≤ 0.0) = P(Z ≤ 0.0)−P(Z ≤ −5.06)
beta2 = 0.50 - 0.00
print("beta2:", beta2)
print("-"*30)

###################################################################

# for 3nd test
population_mean = 52
std = 2.5
n = 16

# beta3 = P(48.81 ≤ X¯ ≤ 51.9)
z31 = (48.81-52)/(2.5/math.sqrt(16))
z32 = (51.9-52)/(2.5/math.sqrt(16))
print("Test3=>")
print("z31:",z31)
print("z32:",z32)

# beta3 = P(−5.10 ≤ Z ≤ -0.16) = P(Z ≤ -0.16)−P(Z ≤ −5.10)
beta3 = 0.0287 - 0.00
print("beta3:", beta3)

print("-"*30)

###################################################################

# for 4th test
population_mean = 52
std = 2.5
n = 16

# beta4 = P(48.42 ≤ X¯ ≤ 51.58)
z41 = (48.42-52)/(2.5/math.sqrt(16))
z42 = (51.58-52)/(2.5/math.sqrt(16))
print("Test4=>")
print("z41:",z41)
print("z42:",z42)

# beta4 = P(−5.73 ≤ Z ≤ -0.67) = P(Z ≤ -0.67)−P(Z ≤ −5.73)
beta4 = 0.2514 - 0.00
print("beta4:", beta4)

Test1=>
z11: -4.427188724235731
z12: -0.6324555320336759
beta1: 0.2643
------------------------------
Test2=>
z21: -5.059644256269407
z22: 0.0
beta2: 0.5
------------------------------
Test3=>
z31: -5.1039999999999965
z32: -0.16000000000000228
beta3: 0.0287
------------------------------
Test4=>
z41: -5.727999999999997
z42: -0.6720000000000027
beta4: 0.2514


###### β at μ=50.5

In [11]:
import math

# for 1st test
μ = 50.5
std = 2.5
n = 10

# beta1 = P(48.5 ≤ X¯ ≤ 51.5)
z11 = (48.5-50.5)/(2.5/math.sqrt(10))
z12 = (51.5-50.5)/(2.5/math.sqrt(10))
print("Test1=>")
print("z11:",z11)
print("z12:",z12)

# beta1 = P(-2.53 ≤ Z ≤ 1.26) = P(Z ≤ 1.27)−P(Z ≤ -2.53)
beta1 = 0.8980 - 0.0057
print("beta1:", beta1)
print("-"*30)

###################################################################

# for 2nd test
population_mean = 50.5
std = 2.5
n = 10

# beta2 = P(48 ≤ X¯ ≤ 52)
z21 = (48-50.5)/(2.5/math.sqrt(10))
z22 = (52-50.5)/(2.5/math.sqrt(10))
print("Test2=>")
print("z21:",z21)
print("z22:",z22)

# beta2 = P(−3.16 ≤ Z ≤ 1.90) = P(Z ≤ 1.90)−P(Z ≤ −3.16)
beta2 = 0.9713 - 0.0008
print("beta2:", beta2)
print("-"*30)

###################################################################

# for 3nd test
population_mean = 50.5
std = 2.5
n = 16

# beta3 = P(48.81 ≤ X¯ ≤ 51.9)
z31 = (48.81-50.5)/(2.5/math.sqrt(16))
z32 = (51.9-50.5)/(2.5/math.sqrt(16))
print("Test3=>")
print("z31:",z31)
print("z32:",z32)

# beta3 = P(-2.70 ≤ Z ≤ 2.24) = P(Z ≤ 2.24)−P(Z ≤ -2.70)
beta3 = 0.9875 - 0.0035
print("beta3:", beta3)

print("-"*30)

###################################################################

# for 4th test
population_mean = 52
std = 2.5
n = 16

# beta4 = P(48.42 ≤ X¯ ≤ 51.58)
z41 = (48.42-50.5)/(2.5/math.sqrt(16))
z42 = (51.58-50.5)/(2.5/math.sqrt(16))
print("Test4=>")
print("z41:",z41)
print("z42:",z42)

# beta4 = P(−3.33 ≤ Z ≤ 1.73) = P(Z ≤ 1.73)−P(Z ≤ −3.33)
beta4 = 0.9582 - 0.0004
print("beta4:", beta4)

Test1=>
z11: -2.5298221281347035
z12: 1.2649110640673518
beta1: 0.8923
------------------------------
Test2=>
z21: -3.1622776601683795
z22: 1.8973665961010278
beta2: 0.9705
------------------------------
Test3=>
z31: -2.703999999999996
z32: 2.2399999999999975
beta3: 0.9840000000000001
------------------------------
Test4=>
z41: -3.327999999999997
z42: 1.7279999999999973
beta4: 0.9578000000000001


#### Conclusion:

* Type I and type II errors are related. When sample size n is fixed, a decrease in the probability of one type of error always results in an increase in the probability of the other
* An increase in sample size will generally reduce both α and β, provided that the critical values are held constant.
* Since we can directly control the probability of wrongly rejecting H0, we always think of rejection of the null hypothesis H0 as a strong conclusion.
* The probability of type II error β is not a constant, but depends on the true value of the parameter and the sample size. It is customary to think of the decision to accept H0 as a weak conclusion, unless we know that β is considerably small.

# Problem Statement 8: 
 
Find the t-score for a sample size of 16 taken from a population with mean 10 when the sample mean is 12 and the sample standard deviation is 1.5. 

In [12]:
#Given parameters
population_mean = 10
sample_mean = 12
std = 1.5
N = 16

#Calculate t-score
import math

t = (sample_mean - population_mean)/(std/math.sqrt(N))
print("t score", t)

t score 5.333333333333333


# Problem Statement 9: 
 
Find the t-score below which we can expect 99% of sample means will fall if samples of size 16 are taken from a normally distributed population. 

###### Find the t-score

In [13]:
#Degree of freedom would be 16-1 = 15
#from t distribution table,for 15 degree of freedom and 0.01 level of significance the t-score is 2.602.

print("t-score =  2.602")

t-score =  2.602


# Problem Statement 10:  
 
If a random sample of size 25 drawn from a normal population gives a mean of 60 and a standard deviation of 4, find the range of t-scores where we can expect to find the middle 95% of all sample means. Compute the probability that (−𝑡0.05 <𝑡<𝑡0.10). 

###### Range of t-scores

In [14]:
#Degree of freedom would be 25-1 = 24
#from t distribution table, for two tailed t-test, 24 degree of freedom and 0.05 level of significance the range of t-scores are as below 

print("Range of t-score:", "-2.064<t<2.064")

Range of t-score: -2.064<t<2.064


###### Probability of (−𝑡0.05 <𝑡<𝑡0.10)

In [15]:
#P(−𝑡0.05 <𝑡<𝑡0.10) = 1-0.05-0.10 = 0.85

print("Probability of (−𝑡0.05 <𝑡<𝑡0.10) is 0.85")

Probability of (−𝑡0.05 <𝑡<𝑡0.10) is 0.85


# Problem Statement 11: 
 
Two-tailed test for difference between two population means<br> 
Is there evidence to conclude that the number of people travelling from Bangalore to Chennai is different from the number of people travelling from Bangalore to Hosur in a week, given the following: 

Population 1: Bangalore to Chennai<br> 
n1 = 1200<br>
x1 = 452<br>
s1 = 212

Population 2: Bangalore to Hosur<br> 
n2 = 800<br>
x2 = 523  
s2 = 185 

###### Solution:

Since n1>30 and n2>30, we will conduct z-test

###### Step-1: State the hypothesis

H0: μ1 – μ2 = 0
    
H0: μ1 – μ2 != 0    

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Setup the decision rule

This is a two-tailed test, using a Z statistic and a 5% level of significance. Reject H0 if Z < -1.960 or is Z > 1.960.

###### Step-4: Calculate the z-score

In [16]:
#Given parameters
n1 = 1200
x1 = 452
s1 = 212
n2 = 800
x2 = 523  
s2 = 185 

import math

z = (x1-x2)/math.sqrt(((s1**2)/n1)+ ((s2**2)/n2))
print("z-score",z)

z-score -7.926428526759299


###### Step-5: Conclusion

since z < -1.960, we reject the null hypothesis.

Therefore, we may conclude that the number of people travelling from Bangalore to Chennai is different from the number of people travelling from Bangalore to Hosur in a week

# Problem Statement 12: 
 
Is there evidence to conclude that the number of people preferring Duracell battery is different from the number of people preferring Energizer battery, given the following: 

Population 1: Duracell<br> 
n1 = 100  
x1 = 308  
s1 = 84 

Population 2: Energizer<br>
n2 = 100<br> 
x2 = 254  
s2 = 67 

###### solution:

Since n1>30 and n2>30, we will conduct z test

###### Step-1: State the hypothesis

H0: μ1 – μ2 = 0
    
H0: μ1 – μ2 != 0

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Setup the decision rule

This is a two-tailed test, using a z statistic and a 5% level of significance. Reject H0 if Z < -1.960 or is Z > 1.960.

###### Step-4: Calculate the z-score

In [17]:
#Given Paraeters
n1 = 100
x1 = 308
s1 = 84
n2 = 100
x2 = 254
s2 = 67

import math

z = (x1-x2)/math.sqrt(((s1**2)/n1)+ ((s2**2)/n2))
print("z-score",z)

z-score 5.025702668336442


###### Step-5: Conclusion

Since z>1.960, we reject the null hypothesis.

Therefore, we may conclude that there is a sufficient evidence that the number of people preferring Duracell battery is different from the number of people preferring Energizer battery.

# Problem Statement 13:  
 
Pooled estimate of the population variance<br>
Does the data provide sufficient evidence to conclude that average percentage increase in the price of sugar differs when it is sold at two different prices?

Population 1:<br>
Price of sugar = Rs. 27.50<br> 
n1 = 14<br> 
x1 = 0.317%<br> 
s1 = 0.12% 

Population 2:<br> 
Price of sugar = Rs. 20.00<br> 
n2 = 9<br> 
x2 = 0.21%<br> 
s2 = 0.11% 

###### Solution:

Since n1<30 and n2<30, we will conduct two tailed t test.

###### Step-1: State the Hypothesis

H0: μ1 – μ2 = 0
    
Ha: μ1 – μ2 != 0

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Find the critical value

degree of freedom,
df = 14+9-2 = 21

for df 21 and level of significance 0.05, 

lower tc value = -2.080<br>
upper tc value = 2.080

###### Step-4: Setup the decision rule

Reject H0 if t < -2.080 or is t > 2.080.

###### Step-5:Calculate t score using pooled t procedure

In [18]:
#Given Parameters
n1 = 14
x1 = 0.317
s1 = 0.12
n2 = 9
x2 = 0.210
s2 = 0.11

import math

#The pooled estimator of the variance
var = (((s1**2)*(n1-1))+ ((s2**2)*(n2-1)))/(n1+n2-2)
var


# t value
t = (x1-x2)/math.sqrt(var*((1/n1)+(1/n2)))
print("t-score:", t)

t-score: 2.15355322387416


###### Step-6: Conclusion

Since t > 2.080, we can reject the null hypothesis.

We may conclude that average percentage increase in the price of sugar is greater when it is sold at Rs. 27.50 than it is when sold for Rs. 20.

# Problem Statement 14:   

The manufacturers of compact disk players want to test whether a small price reduction is enough to increase sales of their product. Is there evidence that the small price reduction is enough to increase sales of compact disk players?

Population 1: Before reduction<br>
n1 = 15<br> 
x1 = Rs. 6598<br> 
s1 = Rs. 844 


Population 2: After reduction<br> 
n2 = 12<br> 
x2 = RS. 6870<br> 
s2 = Rs. 669 

###### Solution:

Since n1<30 and n2<30, we will conduct right-tailed t test to determine whether reducing the price increaces the sales. 

###### Step-1: State the hypothesis

H0: μ1 – μ2 >= 0
    
Ha: μ1 – μ2 < 0

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Find the critical value

degree of freedom,
df = 15+12-2 = 25

for df 25 and level of significance 0.05, the tc value is 1.708.

###### Step-4: Setup the decision rule

Reject H0 if t < -1.708 or is t > 1.708.

###### Step-5: Calculate t score

We assume equal variance of the populations of sales at the two price levels, hence we will use pooled t procedure to calculate t score.

In [19]:
#Given Parameters
n1 = 15
x1 = 6598
s1 = 844
n2 = 12
x2 = 6870
s2 = 669

import math


#The pooled estimator of the variance
var = (((s1**2)*(n1-1))+ ((s2**2)*(n2-1)))/(n1+n2-2)
var


# t value
t = (x1-x2)/math.sqrt(var*((1/n1)+(1/n2)))
print("t-score:", t)

t-score: -0.9098300343990461


###### Step-6: Conclusion

Since the value of t statisctis falls between -1.706 and 1.706, we fail to reject the null hypothesis.

We may conclude that the small price reduction is not enough to increase sales of compact disk players.

# Problem Statement 15:  
 
Comparisons of two population proportions when the hypothesized difference is zero Carry out a two-tailed test of the equality of banks’ share of the car loan market in 1980 and 1995. 

Population 1: 1980<br> 
n1 = 1000<br> 
x1 = 53<br> 
p1 = 0.53 

Population 2: 1985<br> 
n2 = 100<br> 
x2 = 43<br> 
p2 = 0.53

###### Solution:

Since n1>30 and n2>30, we will conduct z test.

###### Step-1: State the hypothesis

H0: p1-p2 = 0
    
Ha: p1-p2 != 0

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Setup the decision rule

This is a two-tailed test, using a Z statistic and a 5% level of significance. Reject H0 if z < -1.960 or is z > 1.960.

###### Step-4: Calculate z score

In [20]:
#Given parameters
n1 = 100
x1 = 53
p1 = 0.53
n2 = 100
x2 = 43
p2 = 0.43


import math

#Combined sample proportion
p = (x1+x2)/(n1+n2)

#z score
z = (p1-p2)/math.sqrt((p*(1-p))*((1/n1)+(1/n2)))
print("z-score:", z)

z-score: 1.4153462926807452


###### Step-5: Conclusion

The z score falls within -1.96 < z < 1.96, hence we fail to reject the null hypothesis.

Even if we use the significance level as 10%, the z statistics falls within non-rejection region. Hence we may conclude that there is a sufficient evidence that the share of banks in the car loan market has changed from 1980 to 1995.

# Problem Statement 16:  
 
Carry out a one-tailed test to determine whether the population proportion of traveler’s check buyers who buy at least $2500 in checks when sweepstakes prizes are offered as at least 10% higher than the proportion of such buyers when no sweepstakes are on. 

Population 1: With sweepstakes<br>
n1 = 300<br> 
x1 = 120<br> 
p1 = 0.40 

Population 2: No sweepstakes<br> 
n2 = 700<br> 
x2 = 140<br>
p2= 0.20 

###### Solution:

Since n1>30 and n2>30, we will conduct z test.

###### Step-1: State the hypothesis

H0: p1-p2 <= 0.10

Ha: p1-p2 > 0.10

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Setup the decision rule

This is a two-tailed test, using a Z statistic and a 5% level of significance. Reject H0 if z < -1.960 or is z > 1.960.

###### Step-4: Calculate z score

In [21]:
#Given parameters
n1 = 300
x1 = 120
p1 = 0.40
n2 = 700
x2 = 140
p2= 0.20 

import math

#Combined sample proportion
p = (x1+x2)/(n1+n2)

#z score
z = ((p1-p2)-0.10)/math.sqrt((p*(1-p))*((1/n1)+(1/n2)))
print("z-score:", z)

z-score: 3.303749523611152


###### Step-5: Conclusion

The z score falls outside -1.96 < z < 1.96, hence we reject the null hypothesis.

Since the value of the test statistic is above the critical point, even for a level of significance as small as 0.001, the null hypothesis may be rejected, and we may conclude that the proportion of customers buying at least $2500 of travelers checks is at least 10% higher when sweepstakes are on.

# Problem Statement 17:  
 
A die is thrown 132 times with the following results: Number turned up: 1, 2, 3, 4, 5, 6 Frequency: 16, 20, 25, 14, 29, 28 Is the die unbiased? Consider the degrees of freedom as p -1. 

###### Solution:

We are going to perform chi-square independence test

###### Step-1: State the hypothesis

H0:The die is unbiased.
    
Ha:The die is not unbiased.

###### Step-2: Determine the confidence interval

Assume the confidence level is 95%

###### Step-3: Find the critical value

In [22]:
from scipy import stats

#degree of freedom
df = 6 -1
df

#Find the critical value
stats.chi2.ppf(0.95,df)

11.070497693516351

###### Step-4: Setup the decision rule

Using the chi suqare statistic and a 5% level of significance. Reject H0 if stat > 11.07.

###### Step-5: Calculate the test Statistics

On the hypothesis that the die is unbiased we should expect the frequency of each number to be 132/6 =22.

In [23]:
from scipy import stats

#Goodness of fit
stats.chisquare([16, 20, 25, 14, 29, 28],[22,22,22,22,22,22])

Power_divergenceResult(statistic=9.0, pvalue=0.1090641579497725)

###### Step-6: Conclusion

Since the calculated value of stat is less than 11.07, we fail to reject the null hypothesis.

We may conclude that the die is unbiased.

# Problem Statement 18:  
 
In a certain town, there are about one million eligible voters. A simple random sample of 10,000 eligible voters was chosen to study the relationship between gender and participation in the last election. The results are summarized in the following 2X2 (read two by two) contingency table:  

We would want to check whether being a man or a woman (columns) is independent of having voted in the last election (rows). In other words, is “gender and voting independent”? 

###### Solution:

We are going to perform chi-square independence test

###### Step-1: State the null hypothesis

H0 = Gender and voting are independent

Ha = Gender and voting are dependent

###### Step-2:Determine the confidence interval

Assume the confidence level is 95%.

###### Step-3: Find the critical value

In [24]:
from scipy import stats

#degree of freedom
df = (2 -1)*(2-1)
df

#Find the critical value
stats.chi2.ppf(0.95,df)

3.841458820694124

###### Step-4: setup the decision rule

Using the chi suqare statistic and a 5% level of significance. Reject H0 if stat > 3.84.

###### Step-5: Find the stats

In [25]:
from scipy import stats

#Contingency table
table = [[2792,3591],[1486,2131]]
print("Contingency table:", table,'\n')

stat, p, dof, expected = stats.chi2_contingency(table)
print("test stat:", stat,'\n')
print("p value:", p,'\n')
print("degree of freedom:", dof,'\n')
print("expected value:\n", expected,'\n')

Contingency table: [[2792, 3591], [1486, 2131]] 

test stat: 6.552337973968181 

p value: 0.01047464310091866 

degree of freedom: 1 

expected value:
 [[2730.6474 3652.3526]
 [1547.3526 2069.6474]] 



###### Step-6: Conclusion

Since calculated value of stat is greater that the critical value(6.552 > 3.84), therefore we reject the null hypothesis.

We may conclude that the gender is dependent of having voted in the last election.

# Problem Statement 19:  
 
A sample of 100 voters are asked which of four candidates they would vote for in an election. The number supporting each candidate is given below: 

Do the data suggest that all candidates are equally popular? [Chi-Square = 14.96, with 3 df, p < 0.05 . 

###### Solution:

We are going to perform chi-square independence test

###### Step-1: State the null hypothesis

H0: all candidates are equally popular.

Ha: all candidates are not equally popular.

###### Step-2:Determine the confidence interval

The given p value is 0.05, hence the confidence interval would be 95%.

###### Step-3: Find the critical value

In [26]:
from scipy import stats

#degree of freedom
df = 4 -1
df

#Find the critical value
stats.chi2.ppf(0.95,df)

7.814727903251179

###### Step-4: setup the decision rule

Using the chi suqare statistic and a 95% confidence level, Reject H0 if stat > 7.81.

###### Step-5:Find the stats

On the hypothesis that the die is unbiased we should expect the frequency of each number to be 100/4 =25.

In [27]:
from scipy import stats

#goodness of fit
stats.chisquare([41,19,24,16],[25,25,25,25])

Power_divergenceResult(statistic=14.959999999999999, pvalue=0.001851152735284203)

###### Step-6:Conclusion

Since the calculated value of stat is greater than 7.81, we can reject the null hypothesis.

We may conclude that all candidates are not equally popular.

# Problem Statement 20: 
Children of three ages are asked to indicate their preference for three photographs of adults. Do the data suggest that there is a significant relationship between age and photograph preference? What is wrong with this study? [Chi-Square = 29.6, with 4 df: 𝑝  < 0.05]. 

###### Solution:

We are going to perform chi-square independence test

###### Step-1: State the hypothesis

H0 = Age and photograph preference are independent

Ha = Age and photograph preference are dependent

###### Step-2:Determine the confidence interval

The given p value is 0.05, hence the confidence interval would be 95%.

###### Step-3: Find the critical value

In [28]:
from scipy import stats

#degree of freedom
df = (3-1)*(3-1)
df

#Find the critical value
stats.chi2.ppf(0.95,df)

9.487729036781154

###### Step-4: setup the decision rule

Using the chi suqare statistic and a 95% confidence level, Reject H0 if stat > 9.48.

###### Step-5:Find the stats

In [29]:
from scipy import stats

#Contingency table
table = [[18,22,20],[2,28,40],[20,10,40]]
print("Contingency table:", table,'\n')

stat, p, dof, expected = stats.chi2_contingency(table)
print("test stat:", stat,'\n')
print("p value:", p,'\n')
print("degree of freedom:", dof,'\n')
print("expected value:\n", expected,'\n')

Contingency table: [[18, 22, 20], [2, 28, 40], [20, 10, 40]] 

test stat: 29.603174603174608 

p value: 5.894582174262606e-06 

degree of freedom: 4 

expected value:
 [[12. 18. 30.]
 [14. 21. 35.]
 [14. 21. 35.]] 



###### Step-6: Conclusion

Since the calculated value of stat is greater than 9.48, we can reject the null hypothesis.

We may conclude that Age and photograph preference are significantly dependent.

# Problem Statement 21:  
 
A study of conformity using the Asch paradigm involved two conditions: one where one confederate supported the true judgement and another where no confederate gave the correct response. 

Is there a significant difference between the "support" and "no support" conditions in the frequency with which individuals are likely to conform? [Chi-Square = 19.87, with 1 df: 𝑝  < 0.05].

###### Solution:

We are going to perform chi-square independence test.

###### Step-1: State the hypothesis

H0 = There is no significant difference between the "support" and "no support" conditions

Ha = There is a significant difference between the "support" and "no support" conditions

###### Step-2:Determine the confidence interval

The given p value is 0.05, hence the confidence interval would be 95%.

###### Step-3: Find the critical value

In [30]:
from scipy import stats

#degree of freedom
df = (2-1)*(2-1)
df

#Find the critical value
stats.chi2.ppf(0.95,df)

3.841458820694124

###### Step-4: Setup the decision rule

Using the chi suqare statistic and a 95% confidence level, Reject H0 if stat > 3.84.

###### Step-5:Find the stats

In [31]:
from scipy import stats

#Contingency table
table = [[18,40],[32,10]]
print("Contingency table:", table,'\n')

stat, p, dof, expected = stats.chi2_contingency(table)
print("test stat:", stat,'\n')
print("p value:", p,'\n')
print("degree of freedom:", dof,'\n')
print("expected value:\n", expected,'\n')

Contingency table: [[18, 40], [32, 10]] 

test stat: 18.10344827586207 

p value: 2.092221924065813e-05 

degree of freedom: 1 

expected value:
 [[29. 29.]
 [21. 21.]] 



###### Step-6: Conclusion

Since the calculated value of stat is greater than 3.84, we can reject the null hypothesis.

Therefore we may conclude that there is a significant difference between the "support" and "no support" conditions, in terms of the frequency with which individuals conformed.

# Problem Statement 22:  
 
We want to test whether short people differ with respect to their leadership qualities (Genghis Khan, Adolf Hitler and Napoleon were all stature-deprived, and how many midget MP's are there?) The following table shows the frequencies with which 43 short people and 52 tall people were categorized as "leaders", "followers" or as "unclassifiable". Is there a relationship between height and leadership qualities?  
[Chi-Square = 10.71, with 2 df: 𝑝  < 0.01]. 

###### Solution:

We are going to perform chi-square independence test.

###### Step-1: State the hypothesis

H0 = There is no relationship between height and leadership qualities.

Ha = There is a significant relationship between height and leadership qualities.

###### Step-2:Determine the confidence interval

The given p value is 0.01, hence the confidence interval would be 99%.

###### Step-3: Find the critical value

In [32]:
from scipy import stats

#degree of freedom
df = (3-1)*(2-1)
df

#Find the critical value
stats.chi2.ppf(0.99,df)

9.21034037197618

###### Step-4: Setup the decision rule

Using the chi suqare statistic and a 99% confidence level, Reject H0 if stat > 9.21.

###### Step-5:Find the stats

In [33]:
from scipy import stats

#Contingency table
table = [[12,32],[22,14],[9,6]]
print("Contingency table:", table,'\n')

stat, p, dof, expected = stats.chi2_contingency(table)
print("test stat:", stat,'\n')
print("p value:", p,'\n')
print("degree of freedom:", dof,'\n')
print("expected value:\n", expected,'\n')

Contingency table: [[12, 32], [22, 14], [9, 6]] 

test stat: 10.712198008709638 

p value: 0.004719280137040844 

degree of freedom: 2 

expected value:
 [[19.91578947 24.08421053]
 [16.29473684 19.70526316]
 [ 6.78947368  8.21052632]] 



###### Step-6: Conclusion

Since the calculated value of stat is greater than 9.21, we can reject the null hypothesis.

Therefore we may conclude that there is a significant relationship between height and leadership qualities.

# Problem Statement 23:  
 
Each respondent in the Current Population Survey of March 1993 was classified as employed, unemployed, or outside the labor force. The results for men in California age 3544 can be cross-tabulated by marital status, as follows: 

Men of different marital status seem to have different distributions of labor force status. Or is this just chance variation? (you may assume the table results from a simple random sample.) 

###### Solution:

We are going to perform chi-square independence test.

###### Step-1: State the hypothesis

H0: The mariatal status and labor force status are independent.
    
Ha: The mariatal status and labor force status are significantly dependent.

###### Step-2:Determine the confidence interval

Assume the significance level is 0.05, hence the confidence inteval would be .095.

###### Step-3: Find the critical value

In [34]:
from scipy import stats

#degree of freedom
df = (3-1)*(3-1)
df

#Find the critical value
stats.chi2.ppf(0.95,df)

9.487729036781154

###### Step-4: Setup the decision rule

Using the chi suqare statistic and 95% confidence level, Reject H0 if stat > 9.48.

###### Step-5:Find the stats

In [35]:
from scipy import stats

#Contingency table
table = [[12,32,114],[22,14,20],[9,6,25]]
print("Contingency table:", table,'\n')

stat, p, dof, expected = stats.chi2_contingency(table)
print("test stat:", stat,'\n')
print("p value:", p,'\n')
print("degree of freedom:", dof,'\n')
print("expected value:\n", expected,'\n')

Contingency table: [[12, 32, 114], [22, 14, 20], [9, 6, 25]] 

test stat: 35.31735051680232 

p value: 3.9977495560257406e-07 

degree of freedom: 4 

expected value:
 [[26.7480315  32.34645669 98.90551181]
 [ 9.48031496 11.46456693 35.05511811]
 [ 6.77165354  8.18897638 25.03937008]] 



###### Step-6: Conclusion

Since the calculated value of stat is greater than 9.48, we can reject the null hypothesis.

Therefore we may conclude that the mariatal status and labor force status significanly dependent, hence men of different marital status seem to have different distributions of labor force status.